apps/messages.py

```diff
@@ -0,0 +1,4 @@
+APPLICATION_TEMPORARILY_INACTIVE = """This application ({}) is temporarily inactive.
+        If you are the app maintainer, please contact the Blue Button 2.0 API team.
+        If you are a Medicare Beneficiary and need assistance,
+        please contact the application support or call 1-800-MEDICARE (1-800-633-4227)"""
```

Comment:
I think this should be moved in to the base.py settings. Maybe at this location:   https://github.com/CMSgov/bluebutton-web-server/blob/8cd9c5a45ce3154284167ae1e90374917f21bb9f/hhs_oauth_server/settings/base.py#L354

It could support an override via a deployment ENV variable too with something similar to:

```python
APPLICATION_TEMPORARILY_INACTIVE_MESG = env('DJANGO_APPLICATION_TEMPORARILY_INACTIVE_MESG', "message text here!")
```

Reply:
good idea